src/libs/actions/Report.js

```diff
@@ -1196,18 +1196,6 @@ function deleteReportComment(reportID, reportAction) {
         },
     ];
 
-    const successData = [
-        {
-            onyxMethod: CONST.ONYX.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
-            value: {
-                [sequenceNumber]: {
-                    pendingAction: null,
-                },
-            },
-        },
-    ];
-
```

Comment:
Just rubber ducking so this change makes sense to me...

>The https://github.com/Expensify/App/pull/10780 introduced a bug where in some cases (usually if the api took some time to delete the message) if you successfully deleted a message it would re-appear for a second and then disappear again.

So by setting the `pendingAction: null` on success we would briefly show the message again because it is seen as "not deleted" otherwise and since it is not `pendingAction === 'delete'` then we would not hide it in `OfflineWithFeedback`?

I think maybe this change will also be fixed by the PR changes we are discussing here https://github.com/Expensify/App/pull/10929#discussion_r973220659

I'll do some testing and report back.

Reply:
So after testing my branch I see that this issue is no longer present. How would you like to proceed?

Reply:
If your P.R. fixes it by that pendingAction check, I think we should dismiss this one and link these issues to it:
https://github.com/Expensify/App/issues/10799
https://github.com/Expensify/App/issues/10753